src/pages/home/report/ReportActionContextMenuItem.js

```diff
@@ -68,6 +68,8 @@ class ReportActionContextMenuItem extends Component {
                 ? (
                     <Tooltip text={text}>
                         <Pressable
+                            accessible
+                            accessibilityLabel={text}
```

Comment:
What about adding some of these other ones like

https://reactnative.dev/docs/view#accessibilityhint

>An accessibility hint helps users understand what will happen when they perform an action on the accessibility element when that result is not clear from the accessibility label.

or 

https://reactnative.dev/docs/view#accessibilityrole

I think this would be a `menuitem` maybe and the `ReportActionContextMenu` itself would probably have `menu` set here?

There's a whole bunch of stuff we could do... what are we prioritizing?

https://reactnative.dev/docs/accessibility

Reply:
I think we just want it to be readable by screen readers for now.

Reply:
Try going to settings, turn on Voiceover in Accessibility and use E.cash on Desktop; it's kind of a horrendous experience.